### ISSUE TITLE
Request Description is cleared on import

### ISSUE THREAD

**Issue by awross-reed** [a85/POSTMan-Chrome-Extension#2117]
Version: 4.3.2
App: Chrome app
OS: Windows 7 Enterprise
Interceptor: No

I export a simple Collection of requests as a V2 Collection. I then import this file and overwrite the original. I expect the description I set for my request to be imported, but it is not.

Exported JSON, original:

```
{
    "variables": [],
    "info": {
        "name": "Test_20160610",
        "_postman_id": "807301c4-6f6e-b16c-1ee5-ace99d0fa087",
        "description": "",
        "schema": "https://schema.getpostman.com/json/collection/v2.0.0/collection.json"
    },
    "item": [
        {
            "name": "DigestAuth Request TEST",
            "event": [
                {
                    "listen": "test",
                    "script": {
                        "type": "text/javascript",
                        "exec": "tests[\"response code is 401\"] = responseCode.code === 401;\ntests[\"response has WWW-Authenticate header\"] = (postman.getResponseHeader('WWW-Authenticate'));\n\nvar authenticateHeader = postman.getResponseHeader('WWW-Authenticate'),\n    realmStart = authenticateHeader.indexOf('\"',authenticateHeader.indexOf(\"realm\")) + 1 ,\n    realmEnd = authenticateHeader.indexOf('\"',realmStart),\n    realm = authenticateHeader.slice(realmStart,realmEnd),\n    nonceStart = authenticateHeader.indexOf('\"',authenticateHeader.indexOf(\"nonce\")) + 1,\n    nonceEnd = authenticateHeader.indexOf('\"',nonceStart),\n    nonce = authenticateHeader.slice(nonceStart,nonceEnd);\n    \npostman.setGlobalVariable('echo_digest_realm', realm);\npostman.setGlobalVariable('echo_digest_nonce', nonce);"
                    }
                }
            ],
            "request": {
                "url": "https://echo.getpostman.com/digest-auth",
                "method": "GET",
                "header": [],
                "body": {
                    "mode": "formdata",
                    "formdata": [
                        {
                            "key": "code",
                            "value": "xWnkliVQJURqB2x1",
                            "type": "text",
                            "enabled": true
                        },
                        {
                            "key": "grant_type",
                            "value": "authorization_code",
                            "type": "text",
                            "enabled": true
                        },
                        {
                            "key": "redirect_uri",
                            "value": "https://www.getpostman.com/oauth2/callback",
                            "type": "text",
                            "enabled": true
                        },
                        {
                            "key": "client_id",
                            "value": "abc123",
                            "type": "text",
                            "enabled": true
                        },
                        {
                            "key": "client_secret",
                            "value": "ssh-secret",
                            "type": "text",
                            "enabled": true
                        }
                    ]
                },
                "description": "Example description goes here"
            },
            "response": []
        }
    ]
}
```

Exported JSON after import:

```
{
    "variables": [],
    "info": {
        "name": "Test_20160610 copy",
        "_postman_id": "9d399078-be21-1bdd-7045-5ed6c605d8c3",
        "description": "",
        "schema": "https://schema.getpostman.com/json/collection/v2.0.0/collection.json"
    },
    "item": [
        {
            "name": "DigestAuth Request TEST",
            "event": [
                {
                    "listen": "test",
                    "script": {
                        "type": "text/javascript",
                        "exec": "tests[\"response code is 401\"] = responseCode.code === 401;\ntests[\"response has WWW-Authenticate header\"] = (postman.getResponseHeader('WWW-Authenticate'));\n\nvar authenticateHeader = postman.getResponseHeader('WWW-Authenticate'),\n    realmStart = authenticateHeader.indexOf('\"',authenticateHeader.indexOf(\"realm\")) + 1 ,\n    realmEnd = authenticateHeader.indexOf('\"',realmStart),\n    realm = authenticateHeader.slice(realmStart,realmEnd),\n    nonceStart = authenticateHeader.indexOf('\"',authenticateHeader.indexOf(\"nonce\")) + 1,\n    nonceEnd = authenticateHeader.indexOf('\"',nonceStart),\n    nonce = authenticateHeader.slice(nonceStart,nonceEnd);\n    \npostman.setGlobalVariable('echo_digest_realm', realm);\npostman.setGlobalVariable('echo_digest_nonce', nonce);"
                    }
                }
            ],
            "request": {
                "url": "https://echo.getpostman.com/digest-auth",
                "method": "GET",
                "header": [],
                "body": {
                    "mode": "formdata",
                    "formdata": [
                        {
                            "key": "code",
                            "value": "xWnkliVQJURqB2x1",
                            "type": "text",
                            "enabled": true
                        },
                        {
                            "key": "grant_type",
                            "value": "authorization_code",
                            "type": "text",
                            "enabled": true
                        },
                        {
                            "key": "redirect_uri",
                            "value": "https://www.getpostman.com/oauth2/callback",
                            "type": "text",
                            "enabled": true
                        },
                        {
                            "key": "client_id",
                            "value": "abc123",
                            "type": "text",
                            "enabled": true
                        },
                        {
                            "key": "client_secret",
                            "value": "ssh-secret",
                            "type": "text",
                            "enabled": true
                        }
                    ]
                }
            },
            "response": []
        }
    ]
}
```

**Comment by abhijitkane** [comment]
@awross-reed Thanks for reporting this. We'll push a fix in the next version of Postman.

**Comment by abhijitkane** [comment]
@awross-reed Can you confirm this fixed in v4.4.1 of Postman (https://www.getpostman.com/docs/updating)?

**Comment by awross-reed** [comment]
Can confirm this is fixed as of v4.4.1
